ChangeLog.d/9302.txt

```diff
@@ -0,0 +1,8 @@
+Features
+   * When the new compilation option MBEDTLS_PSA_STATIC_KEY_SLOTS is enabled,
+     key store uses statically allocated key slot's buffers instead of
```

Comment:
Sorry, under a "new features" section (which is where this will appear) I would expect more of "Added new ..." rather than "When ...". 

`statically allocated key slot's buffers` is clunky - the simplest way to fix that is `statically-allocated key slot buffers` (remove the `'s`)

Reply:
We aren't consistent about present tense vs past tense. The entry for the dynamic key store starts with “When the new compilation option MBEDTLS_PSA_KEY_STORE_DYNAMIC is enabled”, and I think that's fine.